src/web/routes/application/check/get.js

```diff
@@ -27,6 +28,8 @@ const getCheck = (req, res) => {
   const stepArrays = steps.map(getRowData(req))
   const checkRowData = flatten(stepArrays)
 
+  stateMachine.setState(states.IN_REVIEW, req)
```

Comment:
I think this is the wrong place for updating state, but again can be handled in another PR